### ISSUE TITLE
festival - missing dependency in Mint

### ISSUE THREAD

**Issue by ddeath** [DragonComputer/Dragonfire#29]
When I tried to open dragonfire it crashed with:

```
Traceback (most recent call last):
  File "/usr/local/bin/dragonfire", line 11, in <module>
    sys.exit(initiate())
  File "/usr/local/lib/python2.7/dist-packages/dragonfire/__init__.py", line 494, in initiate
    dragon_greet()
  File "/usr/local/lib/python2.7/dist-packages/dragonfire/__init__.py", line 453, in dragon_greet
    userin.say("Good evening " + user_prefix)
  File "/usr/local/lib/python2.7/dist-packages/dragonfire/utilities.py", line 36, in say
    proc = subprocess.Popen(["festival","--tts"], stdin=subprocess.PIPE, stdout=FNULL, stderr=FNULL)
  File "/usr/lib/python2.7/subprocess.py", line 711, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1343, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
```

solution is:

`sudo apt-get install festival`

**Comment by mertyildiran** [comment]
@ddeath agani, it's included in [this line](https://github.com/DragonComputer/Dragonfire/blob/master/setup.py#L22) and [this line](https://github.com/DragonComputer/Dragonfire/blob/master/setup.py#L40). Probably something bad happened on the installation before getting to that stage.

Could you please try to install it with **verbose**(-v) option: `sudo pip install -ve .` and tell me what do you see, any error or warning?

**Comment by mertyildiran** [comment]
Fixed with 56dc3687b4a3f63c2e2b66a186a43a13bae9ad01
